test/plugins/test_ftintitle.py

```diff
@@ -39,19 +43,29 @@ def env() -> Generator[FtInTitlePluginFunctional, None, None]:
 
 def set_config(
     env: FtInTitlePluginFunctional,
-    cfg: dict[str, str | bool | list[str]] | None,
+    cfg: dict[str, ConfigValue] | None,
 ) -> None:
     cfg = {} if cfg is None else cfg
     defaults = {
         "drop": False,
         "auto": True,
         "keep_in_artist": False,
         "custom_words": [],
+        "bracket_keywords": ftintitle.DEFAULT_BRACKET_KEYWORDS.copy(),
     }
     env.config["ftintitle"].set(defaults)
     env.config["ftintitle"].set(cfg)
 
 
+def build_plugin(
```

Comment:
Your unit tests should ideally not depend on the plugin being initialised. That's what functional tests are for.

I think, a good way around this is to define `insert_ft_into_title` as a static method:
```py
    @staticmethod
    def insert_ft_into_title(
        title: str, feat_part: str, keywords: list[str]
    ) -> str:
```

And now you can test it without having to depend on plugin init or needing to manage the config:

```py
assert FtInTitlePlugin.insert_ft_into_title(title, ft_part, keywords) ...
```